### ISSUE TITLE
blktrace does not work with pbench-agent

### ISSUE THREAD

**Issue by aokuliar-rh** [distributed-system-analysis/pbench#2434]
`pbench-agent-0.69.9`

Steps to reproduce:
 1) Add `blktrace` to the default tool set
 2) `pbench-register-tool-set` produces output:

```
[brusic2.example.com]The following options are available:
[brusic2.example.com]
[brusic2.example.com]    --install                 verify this perf tool will work
[brusic2.example.com]
[brusic2.example.com]    --start|stop|postprocess  start/stop/postprocess the data collection
[brusic2.example.com]    -d str|--dir=str          directory to store data collection (required)
[brusic2.example.com]    --devices=str,[str]       the list of block devices to trace (required w/ --start)
[brusic2.example.com]
[brusic2.example.com]    -h|--help                 display this help message
```

**Comment by npalaska** [comment]
Since this is under 0.71, I tried to register `blktrace` tool using `pbench-register-tool` command on pbench-agent 0.71 and it worked fine. I also tried adding `blktrace` to the default tool group in the configuration file and registered using `pbench-register-tool-set` and that worked fine too.

**Comment by aokuliar-rh** [comment]
Retested against pbench-agent-0.71.0-13g7b02500fa.noarch and blktrace works fine.

Since this was 0.69 specific and there won't be any more fixes to 0.69, I am closing this.

Thank you
